vuu-ui/packages/vuu-utils/src/logging-level.ts

```diff
@@ -0,0 +1,5 @@
+// initial code for the use of cookies for logging level
```

Comment:
any particular reason this lives in a file of its own ? Feels like it might as well go in  logging-utils